thrust/detail/config/namespace.h

```diff
@@ -14,24 +14,11 @@
  *  limitations under the License.
  */
 
-
-#pragma once
-
-//functions to support blocking
-
+/*! \namespace thrust
+ *  \brief \p thrust is the top-level namespace which contains all Thrust
+ *         functions and types.
+ */
```

Comment:
I think I'd prefer to have this live in `version.h` as it does today - so maybe get rid of this new file and add this back to `version.h`.